src/policy/v3_policy.cpp

```diff
@@ -85,7 +85,6 @@ std::optional<std::string> PackageV3Checks(const CTransactionRef& ptx, int64_t v
             const auto parent_info = [&] {
                 if (mempool_ancestors.size() > 0) {
                     auto& mempool_parent = *mempool_ancestors.begin();
-                    Assume(mempool_parent->GetCountWithDescendants() == 1);
```

Comment:
was this always erroneous?

Reply:
You're right, it's not necessary. I think all we need is to gate on `ATMPArgs::m_allow_replacement` to not hit it in a multi-testmempoolaccept.